### PR TITLE
Wip

### PR DESCRIPTION
**Pull request by schickling**
- [x] Delete duplicates (png/jpg)
- [x] Delete background images
- [x] Delete signup image
- [x] Set indentation to 2 spaces (almost all changed files involved)
- [ ] Use functional components where possible
- [ ] Avoid fixed px values (use variables instead)

### CODE REVIEW

File: src/components/HomeView/OpenSource.tsx [hunk 1]
```diff
@@ -1,14 +1,57 @@
...
+                            <span> Open Source</span>
+                        </div>
+                        <Heart className={$p.tl}>e</Heart>
+                    </div>
+                </div>
+                <Copy className={cx($p.f20, $p.o50, $p.tc, $p.pb16, $p.selfCenter)}>
+                    I have hinted that I would often jerk poor Queequeg from between the whale and the ship where he would occasionally fall, from the incessant rolling and swaying of both.
+                </Copy>
+                <Learn className={cx($p.flex, $p.justifyCenter, $p.flexWrap, $p.pv60, $p.ph38)}>
+                    <span className={cx($p.mh10, $p.mv10)}>
```
Comment (schickling) [inline src/components/HomeView/OpenSource.tsx]: You probably don't need these wrapping `<span>` tags.

File: src/components/HomeView/SchemaGraph.tsx [hunk 2]
```diff
@@ -1,14 +1,50 @@
...
 
+const Root=styled.div`
+      display: flex;
+      flex-direction: column;
+      align-items: center; 
+      background-color: rgba(0, 0, 0, 0.03);
+      padding: 20px 300px;
+      
+      @media (max-width: ${breakpoints.p1200}px) {
+      padding: 50px 150px;
```
Comment (schickling) [inline src/components/HomeView/SchemaGraph.tsx]: Indentation

File: src/components/HomeView/Features/FeaturesContent.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import * as React from 'react'
+import {$p} from 'graphcool-styles'
+import * as cx from 'classnames'
+
+interface Props {
+    headline: string,
```
Comment (schickling) [inline src/components/HomeView/Features/FeaturesContent.tsx]: Please don't use commas in type definitions

File: src/components/HomeView/Button.tsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import * as React from 'react'
+import styled from 'styled-components';
+
+const Button = styled.button`
```
Comment (schickling) [inline src/components/HomeView/Button.tsx]: Any reason why you're not using particles here?

File: src/components/HomeView/Features/Features.tsx [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+import { $p } from 'graphcool-styles'
+import styled from 'styled-components'
+import { breakpoints } from '../../../utils/constants'
+import FeaturesContent from './FeaturesContent'
+
+export default class Features extends React.Component<{}, {}> {
+
+  render() {
+    const Container= styled.div`
+      display: flex;
```
Comment (schickling) [inline src/components/HomeView/Features/Features.tsx]: Use fragments

File: src/components/HomeView/OpenSource.tsx (hunk 1 above)
Comment (dulik06) [inline src/components/HomeView/OpenSource.tsx]: Indeed

File: src/components/HomeView/Button.tsx (hunk 4 above)
Comment (dulik06) [inline src/components/HomeView/Button.tsx]: Changed. Button done with $p. Same as SignUp button in the header

**Pull request merged**